app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +31,26 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            setButonLable(text);
+
+            if (text == "something") {
+                setButonLable("something");
+            }
+        } catch (Exception e) {
+        }
+        findViewById(R.id.buton).setOnClickListener(new OnClickListener() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/39d86fb759bdab1db76b818f1155db0332b36239/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L44"></span>The class com.mathroule.demoapp.MainActivity$2 could be refactored into a named <em>static</em> inner class</td>
    </tr>
  </tbody>
</table>